src/test/java/com/googlecode/download/maven/plugin/internal/WGetTest.java

```diff
@@ -48,29 +80,195 @@ public void setUp() throws Exception {
         setVariableValueToObject(mojo, "retries", 1);
         setVariableValueToObject(mojo, "settings", new Settings());
         setVariableValueToObject(mojo, "buildContext", buildContext);
+
+        initializer.accept(mojo);
+        return mojo;
     }
 
     private void cleanUp() {
```

Comment:
@ajarmoniuk can we annotate this method with `@After` and avoid try-finally in every test?
Or we could do a cleanup before setup. It simplifies development as you can check temp dir visually in case of unexpected test failure